lp-steam/templates/csgo/server.j2

```diff
@@ -42,4 +62,27 @@ sv_minupdaterate "{{ item.value.tickrate | default('128') }}"
 sv_minrate 80000
 sv_maxrate 0
 
-exec rules_{{ item.value.rules | default('esl5on5') }}.cfg
+{% if item.value.backup | default(false) %}
```

Comment:
Nice, but this should be organized into subfolders. `mp_backup_round_file` can contain slashes and can be used to accomplish this. I have a solution in mind.